test/parallel/test-repl-tab-complete.js

```diff
@@ -196,6 +198,33 @@ testMe.complete('require(\'n', common.mustCall(function(error, data) {
   });
 }));
 
+{
+  function maybeMkdir(fp) {
+    try {
+      fs.mkdirSync(fp);
+    } catch (err) {
+      if (err.code !== 'EEXIST') throw err;
+    }
+  }
+
+  const node_modules = path.join(process.cwd(), 'node_modules');
+  maybeMkdir(node_modules);
+  const scope = path.join(node_modules, '@nodejsscope');
+  maybeMkdir(scope);
+
+  putIn.run(['.clear']);
+  testMe.complete('require(\'@nodejs', common.mustCall((err, data) => {
+    assert.strictEqual(err, null);
+    assert.deepStrictEqual(data, [['@nodejsscope/'], '@nodejs']);
+    common.rimrafSync(scope);
+    // Only remove node_modules directory if there is nothing in it.
+    const files = fs.readdirSync(node_modules).filter((file) => {
+      return fs.statSync(file).isDirectory()
+    });
+    if (!files.length) common.rimrafSync(node_modules);
+  }));
+}
```

Comment:
100% agree. I'll try and get that working. That should simplify a lot of this test. Thanks!